OpenRA.Game/ModData.cs

```diff
@@ -162,10 +166,26 @@ public List<MiniYamlNode>[] GetRulesYaml()
 			return Manifest.Rules.Select(s => MiniYaml.FromStream(DefaultFileSystem.Open(s), s, stringPool: stringPool)).ToArray();
 		}
 
+		public SequenceSet GetSequences(IReadOnlyFileSystem fileSystem, string tileset, MiniYaml sequenceDefinitions)
+		{
+			if (sequences == null || this.tileset != tileset
+				|| (sequenceDefinitions == null ^ this.sequenceDefinitions == null)
+				|| (sequenceDefinitions != null && !sequenceDefinitions.Matches(this.sequenceDefinitions)))
```

Comment:
The code in the if branch doesn't make sense when `sequenceDefinitions` is null, so this should be enough:
```suggestion
				|| (sequenceDefinitions != null && !sequenceDefinitions.Matches(this.sequenceDefinitions)))
```

Reply:
it needs to handle the case where `sequenceDefinitions` is null but `this.sequenceDefinitions` isn't.

Changes the function to make it a bit more readable

Reply:
Ok, I thought this wouldn't happen since we would overwrite `this.sequenceDefinitions` with `null` then, which doesn't seem desirable?

Reply:
it's supposed to update to the latest sequences, and if the latest sequences don't have any overwrites it needs to save that it doesn't have any overwrites

Reply:
Ah makes sense, thanks.